packages/error-boundry-fallback/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@opensrp/error-boundry-fallback",
+  "version": "0.0.1",
+  "types": " dist/types",
+  "description": "Error boundry fallback component",
+  "main": "dist/index.js",
+  "repository": "https://github.com/opensrp/web",
+  "author": "Ona Engineering",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/opensrp/web/issues"
+  },
+  "scripts": {
+    "jest": "jest --coverage --verbose --color",
+    "tsc": "tsc",
+    "transpile": "babel src -d dist --root-mode upward --extensions '.ts,.tsx'  --ignore '**/*.test.ts,**/*.test.tsx,**/tests,**/__tests__'"
+  },
+  "files": [
+    "dist"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "jest": {
+    "setupFiles": [
+      "../../setupTests"
+    ]
+  },
+  "peerDependencies": {
```

Comment:
Should we include `antd` in the dependencies or peer dependencies?

Reply:
@moshthepitt what do you think about this?

Reply:
what we need is a formalized description of the properties that should be met for a npm package to either be in peerdependencies, or devdependencies or dependencies. I think this should help us with such decisions henceforth.

I am currently conflicted on what the guiding factors should be:

in the meantime here is an interesting read: https://nodejs.org/es/blog/npm/peer-dependencies/

Reply:
I usually think of it this way:

- **dependencies** - things that we directly depend on.  Remember that these will be downloaded when your package is installed and count to its "weight" e.g. the way the [Gatekeeper package](https://github.com/onaio/js-tools/blob/master/packages/gatekeeper/package.json) depends on client-oauth2 and @onaio/session-reducer
- **peerdependencies** - things that we are building for/against but we expect are already installed in the places where our packages will be used e.g. React is usually a peer dependency when building React libraries; Bootstrap is usually a peer dependency when building things that work with Bootstrap (e.g. reactstrap) but a dependency when building an app that uses Bootstrap (e.g. your web app that uses Bootstrap styling)
- **devdependencies** - things that are only required when doing dev work e.g. testing libraries, type definitions, mocking libraries etc.  You don't expect users of your package to ever interact with these unless they are developers enhancing or working with your package

The above has served me well over the years.  But we should note that:

1. It isn't a list of strict hard and fast rules, more like a guide since some things have to be on a case-by-case basis e.g. sometimes using Lerna things that would be peerdependencies might be devdependencies or when using Sapper everything should be a devdependency, etc.
2. I might be wrong and if I am we should update the above :)

Reply:
@moshthepitt @p-netm @HaseemUlHaq Ok...There is no reference of `antd` in the package.json neither in the `peerDependencies` or the `dependencies`, should it be added in the dependencies?

Reply:
peer dependencies.